showcase/gapic-showcase/src/test/resources/META-INF/native-image/resource-config.json

```diff
@@ -1,3 +1,6 @@
 {
-  "resources":[{"pattern": ".*.json"}]
+  "resources": [
+    { "pattern": ".*.json" },
+    { "pattern": ".*.pem" }
```

Comment:
If I'm understanding correctly, only this change is required for native image testing? Are the changes in ITGdch required or refactoring?

Reply:
The `ITGdch` changes are required. The previous logic created a Path reference to a bundled resource. For normal testing, this path is resolved to the `/target/classes` directory -- but during native image testing, we can't create a Path reference to a resource located inside the native binary. Instead, we have to copy it out to location on the file system and create a path to that copy.

Reply:
I see, thanks!